services/ux-backend/handlers/onboardingtokens/handler.go

```diff
@@ -16,14 +18,35 @@ const (
 func HandleMessage(w http.ResponseWriter, r *http.Request, tokenLifetimeInHours int) {
 	switch r.Method {
 	case "POST":
-		handlePost(w, tokenLifetimeInHours)
+		type Message struct {
+			Quota string `json:"quota"`
+		}
+		var quota Message
+		err := json.NewDecoder(r.Body).Decode(&quota)
+		if err != nil {
+			if err.Error() == "EOF" {
+				handlePost(w, tokenLifetimeInHours, nil)
+				return
+			}
+			http.Error(w, err.Error(), http.StatusBadRequest)
+			return
+		}
+
+		quantity, err := resource.ParseQuantity(quota.Quota)
```

Comment:
`quota.Quota` will be empty for older clients which breaks backward compat unless `resource.ParseQuantity` resistant to empty strings. is mandating it a requirement during onboard?

Reply:
This change is while generation of Token only. Older clients are already connected. they don't need to worry about quota for them.

Reply:
yes, good one. resolve.